README.md

```diff
@@ -6,6 +6,16 @@ SmoothMQ lets you run a private SQS instance on any cloud.
 
 <!-- A drop-in replacement for SQS designed for great developer experience and efficiency. -->
 
+## Quick Start
```

Comment:
There is already a `Getting Started` and `Running` section in the readme - maybe `Docker Compose` could be a subheading there.

Reply:
I don't have this on a branch so edits on this are hard.